stdlib/3/time.pyi

```diff
@@ -5,10 +5,17 @@
 # see: http://nullege.com/codes/search?cq=time
 
 import sys
-from typing import Tuple, Union
+from typing import Any, NamedTuple, Tuple, Union
 from types import SimpleNamespace
 
-TimeTuple = Tuple[int, int, int, int, int, int, int, int, int]
+if sys.version_info >= (3, 3):
+    TimeTuple = Union[
```

Comment:
In what sense is it overly restrictive? All functions that take a TimeTuple also take a struct_time; IMO you should always use the latter when specifying the extra values. The struct_time, when unpacked, always returns exactly 9 values (even if it stores extra values).

The docs are pretty vague; the best I can find is here: https://docs.python.org/3/library/time.html?highlight=mktime#time.mktime -- this still says 9-tuple. Also note the definition of https://docs.python.org/3/library/time.html?highlight=mktime#time.struct_time which says that tm_gmtoff and tm_zone are only available on platforms where the C library supports these.